From 23362b1a7df71dfa25f43a2c54d8035638edb2c7 Mon Sep 17 00:00:00 2001
From: Cosmin-Gabriel Samoila <cosmin.samoila@nxp.com>
Date: Wed, 21 Nov 2018 11:20:33 +0200
Subject: [PATCH 5184/5242] MLKU-24-10 sound: asoc: change License comment to
 SPDX format

commit  556c040708e10c506577c965609ba9092892405a from
https://source.codeaurora.org/external/imx/linux-imx.git

Change source and header License comment to SPDX format.

Signed-off-by: Cosmin-Gabriel Samoila <cosmin.samoila@nxp.com>
Reviewed-by: Daniel Baluta <daniel.baluta@nxp.com>
(cherry picked from commit 6d17e38a8d8cd3eab3015988a6f24cf64ec0d1df)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/fsl_micfil.c |   12 ++----------
 sound/soc/fsl/fsl_micfil.h |   10 +++-------
 2 files changed, 5 insertions(+), 17 deletions(-)

diff --git a/sound/soc/fsl/fsl_micfil.c b/sound/soc/fsl/fsl_micfil.c
index fe821a9..e84f030f 100644
--- a/sound/soc/fsl/fsl_micfil.c
+++ b/sound/soc/fsl/fsl_micfil.c
@@ -1,13 +1,5 @@
-/*
- * Copyright 2018 NXP
- *
- * The code contained herein is licensed under the GNU General Public
- * License. You may obtain a copy of the GNU General Public License
- * Version 2 or later at the following locations:
- *
- * http://www.opensource.org/licenses/gpl-license.html
- * http://www.gnu.org/copyleft/gpl.html
- */
+// SPDX-License-Identifier: GPL-2.0
+// Copyright 2018 NXP
 
 #include <linux/atomic.h>
 #include <linux/clk.h>
diff --git a/sound/soc/fsl/fsl_micfil.h b/sound/soc/fsl/fsl_micfil.h
index c4a98bd..792187b 100644
--- a/sound/soc/fsl/fsl_micfil.h
+++ b/sound/soc/fsl/fsl_micfil.h
@@ -1,11 +1,7 @@
+/* SPDX-License-Identifier: GPL-2.0 */
 /*
- * fsl_micfil.h - PDM Microphone Interface for the NXP i.MX SoC
- *
- * Copyright (C) 2018 NXP
- *
- * This file is licensed under the terms of the GNU General Public License
- * version 2. This program is licensed "as is" without any warranty of any
- * kind, whether express or implied.
+ * PDM Microphone Interface for the NXP i.MX SoC
+ * Copyright 2018 NXP
  */
 
 #ifndef _FSL_MICFIL_H
-- 
1.7.9.5

